### PR TITLE
Automatic recovery from failed nodes - Reactive(IPI)

### PR DESCRIPTION
**Pull request by PrasadDesala**
Signed-off-by: Prasad Desala <tdesala@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/node.py [hunk 1]
```diff
@@ -359,3 +364,51 @@ def get_node_resource_utilization_from_oc_describe(nodename=None, node_type='wor
...
+        list: App pod running node names
+    """
+    app_pod_node_name = []
+    for node in pod_obj:
+        app_pod_node_name.append(pod.get_pod_node(node).name)
+    return app_pod_node_name
+
+
+def get_both_osd_and_app_pod_running_node(
+        osd_running_nodes, app_pod_running_nodes
```
Comment (ebenahar) [inline ocs_ci/ocs/node.py]: delete extra indentation

**Comment by PrasadDesala** [comment]
> PR validation [Job UNSTABLE](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1223//testReport) (some or all tests failed).

This PR validation seems failing because of https://bugzilla.redhat.com/show_bug.cgi?id=1810014

File: ocs_ci/ocs/node.py (hunk 1 above)
Comment (PrasadDesala) [reply ocs_ci/ocs/node.py]: Done

**Comment by ebenahar** [comment]
@PrasadDesala please rebase

**Comment by PrasadDesala** [comment]
> @PrasadDesala please rebase

Done. Also added WA #1605

**Comment by vasukulkarni** [comment]
@ebenahar @RazTamir please review

**Pull request merged**